jans-config-api/plugins/admin-ui-plugin/src/main/java/io/jans/ca/plugin/adminui/service/license/LicenseDetailsService.java

```diff
@@ -363,6 +382,14 @@ public GenericResponse generateTrialLicense() {
             log.error("{}: {}", TRIAL_GENERATE_ERROR_RESPONSE, jsonData);
             return CommonUtils.createGenericResponse(false, response.getStatus(), "Error in generating trial license.");
         } catch (Exception e) {
+            if (response.getStatus() == 404) {
+                log.error("{}", LICENSE_APIS_404);
+                return CommonUtils.createGenericResponse(false, response.getStatus(), LICENSE_APIS_404);
+            }
+            if (response.getStatus() == 503) {
+                log.error("{}", LICENSE_APIS_503);
+                return CommonUtils.createGenericResponse(false, response.getStatus(), LICENSE_APIS_503);
+            }
```

Comment:
I'm approving this PR since it's not critical but in future it makes sense to extract it into separate method to not have copy/paste.